src/domain/participant/index.js

```diff
@@ -447,16 +447,53 @@ const getLimitsForAllParticipants = async ({ currency = null, type = null }) =>
  * @returns {integer} - Returns number of database rows affected if successful, or throws an error if failed
  */
 
+// {
+//   "currency": "USD",
+//   "limit": {
+//       "type": "NET_DEBIT_CAP",
+//       "value": 1500,
+//       "thresholdAlarmPercentage": 10
+//   }
+// }
+
 const adjustLimits = async (name, payload) => {
   try {
-    const participant = await ParticipantFacade.getByNameAndCurrency(name, payload.currency, Enum.LedgerAccountType.POSITION)
+    let { limit, currency } = payload
+    payload.name = name
```

Comment:
I'd move this line just before line 466, where payload is used, not because it is bad in the beginning, but on the previous line you have payload object decomposing, and then you add other properties.